lib/stdlib/test/edlin_expand_SUITE.erl

```diff
@@ -18,12 +18,17 @@
 %% %CopyrightEnd%
```

Comment:
Can you add tests for Unicode names, filenames, fields variables etc? Just in case anyone runs [beamoji](https://github.com/spawnfest/beamoji) on the source code and tries to autocomplete.